mcodingbot/plugins/info.py

```diff
@@ -9,22 +9,20 @@
 
 @plugin.include
 @crescent.command(name="ping", description="Pong!")
-class PingCommand:
-    async def callback(self, ctx: Context) -> None:
-        await ctx.respond(f"Pong! {round(ctx.app.heartbeat_latency*1000)} ms.")
+async def ping(ctx: Context) -> None:
+    await ctx.respond(f"Pong! {round(ctx.app.heartbeat_latency*1000)} ms.")
 
 
 @plugin.include
 @crescent.command(name="links", description="Useful links")
-class LinksCommand:
-    async def callback(self, ctx: Context) -> None:
-        embed = hikari.Embed(
-            title="Links",
-            color=CONFIG.theme,
-            description=(
-                f"**[mCoding Youtube]({CONFIG.mcoding_youtube})**\n"
-                f"**[mCoding repo]({CONFIG.mcoding_repo})**\n"
-                f"**[mCodingBot repo]({CONFIG.mcodingbot_repo})**"
-            ),
-        )
-        await ctx.respond(embed=embed)
+async def links(ctx: Context) -> None:
+    embed = hikari.Embed(
+        title="Links",
+        color=CONFIG.theme,
+        description=(
+            f"**[mCoding Youtube]({CONFIG.mcoding_youtube})**\n"
+            f"**[mCoding repo]({CONFIG.mcoding_repo})**\n"
```

Comment:
Just a thought, wouldn't be better to write `repository` in full?
```suggestion
            f"**[mCoding repository]({CONFIG.mcoding_repo})**\n"
```

Reply:
Outside scope

Reply:
That's a very small change though

Reply:
Then pr it after. This PR modifies how three functions are written. It will not do anything else.

Reply:
Adding more to this PR will a) make reviews harder and b) add other issues that need to be resolved making it take longer to merge

Reply:
alright alright